src/AWSS3.jl

```diff
@@ -427,7 +427,7 @@ Returns an iterator of `Dict`s with keys `Key`, `LastModified`, `ETag`, `Size`,
 `Owner`, `StorageClass`.
 """
 function s3_list_objects(aws::AbstractAWSConfig, bucket, path_prefix=""; delimiter="/", max_items=nothing, kwargs...)
-    return Channel() do chnl
+    return Channel(ctype=LittleDict, csize=128) do chnl
```

Comment:
Using a sized `Channel` does seem to improve performance slightly. 128 seemed small enough to be fine?

Reply:
sounds fine to me